src/newgrf_engine.cpp

```diff
@@ -704,6 +704,12 @@ static uint32 VehicleGetVariable(Vehicle *v, const VehicleScopeResolver *object,
 			return ret;
 		}
 
+		case 0x63: { // Poweredness relative to arbitrary railtype
+			if (v->type != VEH_TRAIN) return 0;
```

Comment:
Should this be extended for NRT? Roadtypes can also be powered as far as I know.

Reply:
Road/tramtypes don't distinguish powered and compatible, though.

Reply:
IIRC the use-case for this is: A vehicle is dual-powered, it is powered on normal rail, but on electrified rail it raises a pantograph.
The same use-case is valid for roadtypes.

Reply:
i now added the road-catenary flag to var 0x4A, however, i don't see how this would apply to var 0x63, other than returning "1" all the time.